### PR TITLE
embed template

### PR DESCRIPTION
**Pull request by ifochka9519**
Embed template and additional elements

### CODE REVIEW

File: src/main/resources/templates/elements.html [hunk 1]
```diff
@@ -0,0 +1,708 @@
...
+					</p>
+				</div>
+			</section>
+			<!-- End Sample Area -->
+			<!-- Start Button -->
+			<section class="button-area">
+				<div class="container border-top-generic">
+					<h3 class="text-heading">Sample Buttons</h3>
+					<div class="button-group-area">
+						<a th:href="@{#}" class="genric-btn default">Default</a>
```
Comment (inventorsoft-academy) [inline src/main/resources/templates/elements.html:103]: you need to wrap only relative links inside th:href="@{}" construction. if the link is static you can use simple href property

File: src/main/java/com/geekschool/controllers/IndexController.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+@Controller
+@RequestMapping(value = "/")
+public class IndexController {
+    @GetMapping
+    static String getIndex() {
+        return "blog-home";
+    }
+
+    @GetMapping(value = "/elements")
```
Comment (inventorsoft-academy) [inline src/main/java/com/geekschool/controllers/IndexController.java:15]: What is elements?

**Review by inventorsoft-academy — CHANGES_REQUESTED** [review]

**Comment by inventorsoft-academy** [comment]
You don't need to fork this project anymore. You can clone it and create branches right inside this repository

File: src/main/java/com/geekschool/controllers/IndexController.java (hunk 2 above)
Comment (ifochka9519) [reply src/main/java/com/geekschool/controllers/IndexController.java:15]: This is link for page with another elements from this template.

File: src/main/resources/templates/elements.html (hunk 1 above)
Comment (ifochka9519) [reply src/main/resources/templates/elements.html:103]: ok, I fixed

**Pull request closed without merging**